saleor/graphql/order/bulk_mutations/draft_orders.py

```diff
@@ -51,6 +52,30 @@ def get_channel_ids(cls, instances) -> Iterable[Union[UUID, int]]:
         """Get the instances channel ids for channel permission accessible check."""
         return [order.channel_id for order in instances]
 
+    @classmethod
+    def perform_mutation(  # type: ignore[override]
+        cls, _root, info: ResolveInfo, /, *, ids, **data
+    ) -> tuple[int, Optional[ValidationError]]:
+        # code relies on the fact that both protected Models have FK named as `order`
```

Comment:
It is possible to deal with that problem - I can introduce method on User model sth like:
``` python
def get_protected_relations(cls):
    protected_relations = []
    for field in cls._meta.get_fields(include_hidden=True):
        if hasattr(field, "on_delete") and field.on_delete.__name__ == "PROTECT":
            protected_relations.append(field.related_model)
    return protected_relations
```
And then introduce another method to iterate through them and find exact name that is relating to `Order`. But not sure if we should be that careful here, any opinions?